src/test/java/com/squareup/kotlinpoet/FileSpecTest.kt

```diff
@@ -758,4 +758,20 @@ class FileSpecTest {
       |
       |""".trimMargin())
   }
+
+  @Test fun constProperty() {
+    val text = "This is a long string with a newline\nin the middle."
+    val spec = FileSpec.builder("testsrc", "Test")
+        .addProperty(PropertySpec.builder("FOO", String::class, KModifier.CONST)
+            .initializer("%S", text)
+            .build())
+        .build()
+    assertThat(spec.toString()).isEqualTo("""
+      |package testsrc
+      |
+      |import kotlin.String
+      |
+      |const val FOO: String = "${text.replace("\n", "\\n")}"
```

Comment:
I think we'd prefer to see the string here directly. Right now the test relies on the implementation and `replace` and we only want it to depend on the implementation.